velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -915,6 +985,73 @@ bool AggregationFuzzer::verifySortedAggregation(
   return resultOrError.exceptionPtr != nullptr;
 }
 
+bool AggregationFuzzer::verifyDistinctAggregation(
+    const std::vector<std::string>& groupingKeys,
+    const std::vector<std::string>& aggregates,
+    const std::vector<std::string>& masks,
+    const std::vector<RowVectorPtr>& input,
+    bool customVerification,
+    const std::vector<std::shared_ptr<ResultVerifier>>& customVerifiers) {
+  const auto firstPlan = PlanBuilder()
+                             .values(input)
+                             .singleAggregation(groupingKeys, aggregates, masks)
+                             .planNode();
+
+  if (customVerification) {
+    const auto& aggregationNode =
+        std::dynamic_pointer_cast<const core::AggregationNode>(firstPlan);
+    for (auto i = 0; i < customVerifiers.size(); ++i) {
+      auto& verifier = customVerifiers[i];
+      if (verifier == nullptr) {
+        continue;
+      }
+
+      verifier->initialize(
+          input,
+          groupingKeys,
+          aggregationNode->aggregates()[i],
+          aggregationNode->aggregateNames()[i]);
+    }
+  }
+
+  SCOPE_EXIT {
+    if (customVerification) {
+      resetCustomVerifiers(customVerifiers);
+    }
+  };
+
+  // Create all the plans upfront.
+  std::vector<PlanWithSplits> plans;
```

Comment:
nit: Add `firstPlan` to `plans` so that you don't need to pass it additionally to compareEquivalentPlanResults().